[2025-04-13T21:09:40.227+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T21:09:40.250+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__2005-01-01T00:00:00+00:00 [queued]>
[2025-04-13T21:09:40.261+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__2005-01-01T00:00:00+00:00 [queued]>
[2025-04-13T21:09:40.262+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-13T21:09:40.277+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_data> on 2005-01-01 00:00:00+00:00
[2025-04-13T21:09:40.285+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gcs_dlt_data_pipeline', 'process_data', 'scheduled__2005-01-01T00:00:00+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/first_stage/first_stage.py', '--cfg-path', '/tmp/tmpyh8mp3ai']
[2025-04-13T21:09:40.288+0000] {standard_task_runner.py:105} INFO - Job 295: Subtask process_data
[2025-04-13T21:09:40.291+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=289) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-13T21:09:40.292+0000] {standard_task_runner.py:72} INFO - Started process 297 to run task
[2025-04-13T21:09:40.345+0000] {task_command.py:467} INFO - Running <TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__2005-01-01T00:00:00+00:00 [running]> on host 3330be1269a6
[2025-04-13T21:09:40.455+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_dlt_data_pipeline' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2005-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2005-01-01T00:00:00+00:00'
[2025-04-13T21:09:40.457+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T21:10:02.133+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-04-13T21:10:02.164+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T21:10:02.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
